ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -220,92 +240,26 @@
 {% if evpn_rt_type is defined and evpn_rt_type is not none %}
 {%     if evpn_rt_type.admin_subfield is defined and evpn_rt_type.admin_subfield is not none %}
 {%         if evpn_rt_type.admin_subfield == "leaf_asn" %}
-{%             set leaf.evpn_rt_admin_subfield = leaf.bgp_as %}
+{%             set leaf.evpn_rt_admin_subfield = switch.bgp_as %}
 {%         elif evpn_rt_type.admin_subfield == "spine_asn" %}
-{%             set leaf.evpn_rt_admin_subfield = spine.bgp_as %}
+{%             set leaf.evpn_rt_admin_subfield = switch.bgp_as %}
```

Comment:
I believe this should remain `spine.bgp_as`

Reply:
Good catch, fixed. Thank you.